airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionForm.tsx

```diff
@@ -197,6 +197,36 @@ const ConnectionForm: React.FC<ConnectionFormProps> = ({
       {({ isSubmitting, setFieldValue, isValid, dirty, resetForm, values }) => (
         <FormContainer className={className}>
           <FormChangeTracker changed={dirty} formId={formId} />
+          <Section title={null}>
```

Comment:
This could just be a `<StyledSection>` since the title is unused. Or title could be made optional.

Reply:
Editing this does not update the page title. Not sure I'd consider that a blocker, but worth noting.

Reply:
Hey didn't get the last comment

Reply:
@harshithmullapudi There's a component in this file called `<StyledSection />` that is similar to this where you don't need to pass a `title` prop. The feedback is to replace this line with that component.